src/common/src/row.rs

```diff
@@ -14,7 +14,7 @@
 
 /// `CompactedRow` is used in streaming executors' cache, which takes less memory than `Vec<Datum>`.
 /// Executors need to serialize Row into `CompactedRow` before writing into cache.
-#[derive(Clone, Debug)]
```

Comment:
> Default, PartialEq, Eq, Hash

What do we need these for?

Reply:
Oops, sorry for ignoring this comment before merging. I also feel like it's not necessary in most cases. Not sure if we have some places where we directly compare these rows.